src/payment/payment.module.ts

```diff
@@ -81,7 +82,7 @@ export class PaymentModuleImpl implements PaymentModule {
 
   private readonly options: Required<PaymentModuleOptions> = {
     driver: "erc20",
-    network: "holesky",
+    network: EnvUtils.getPaymentNetwork() || "holesky",
```

Comment:
`EnvUtils.getPaymentNetwork` and `EnvUtils.getYagnaSubnet` already have defaults if env is empty, no need for the `|| "holesky"` and `|| "public"`

Reply:
It's better to have defaults in one place